### PR TITLE
Adding support for AT_SEQUENCE_NUMBER and AFTER_SEQUENCE_NUMBER

### PR DESCRIPTION
**Pull request by danguer**
Adding support on DynamoDB Streams for AT_SEQUENCE_NUMBER
 and AFTER_SEQUENCE_NUMBER ShardIteratorType

Change SequenceNumber type to string instead of int to match documentation

### CODE REVIEW

**Comment by danguer** [comment]
Sorry, I didn't create an issue before. 
I found that AT_SEQUENCE_NUMBER and AFTER_SEQUENCE_NUMBER weren't implemented in dynamodb streams so tried to add support for it
Also found the SequenceNumber returned are int compared to string on the documentation for boto / REST API

**Comment by spulec** [comment]
Code looks good! If we can get the tests passing, I'll be happy to merge.

**Comment by danguer** [comment]
I've updated master branch and made changes to support python 2.7 so now the unit test are good

File: moto/dynamodbstreams/responses.py [hunk 1]
```diff
@@ -23,8 +23,13 @@ def get_shard_iterator(self):
         arn = self._get_param('StreamArn')
         shard_id = self._get_param('ShardId')
         shard_iterator_type = self._get_param('ShardIteratorType')
+        sequence_number = self._get_param('SequenceNumber')
+        # according to documentation sequence_number param should be string
+        if isinstance(sequence_number, "".__class__):
```
Comment (spulec) [inline moto/dynamodbstreams/responses.py]: Can we use [six.string_types](https://six.readthedocs.io/#six.string_types) here?

**Comment by spulec** [comment]
Looks great; thanks!

**Pull request merged**